components/raftstore/src/store/peer.rs

```diff
@@ -844,12 +846,11 @@ pub enum UnsafeRecoveryState {
 // it is checked every time this peer applies a new entry or a snapshot,
 // if the latest committed index is met, the syncer will be called to notify the
 // result.
-#[derive(Debug)]
-pub struct FlashbackState(Option<Sender<bool>>);
+pub struct FlashbackMemoryState(Option<Sender<bool>>);
```

Comment:
There is no need to emphasize that this is a memory state since it will only be set in memory, let's keep the original name.